velox/substrait/tests/VeloxSubstraitRoundTripPlanConverterTest.cpp

```diff
@@ -93,6 +93,16 @@ TEST_F(VeloxSubstraitRoundTripPlanConverterTest, filter) {
   assertPlanConversion(plan, "SELECT * FROM tmp WHERE c2 < 1000");
 }
 
+TEST_F(VeloxSubstraitRoundTripPlanConverterTest, null) {
+  RowVectorPtr vectors = makeRowVector(
+      {makeFlatVector<int32_t>({3, 2}),
+       makeFlatVector<int32_t>(2, nullptr, nullEvery(1))});
+  createDuckDbTable({vectors});
+
+  auto plan = PlanBuilder().values({vectors}).project({"NULL"}).planNode();
```

Comment:
Looks like this plan doesn't use any of the output of the values node. Hence, you can simplify the plan by using empty vectors.

```
auto vectors = makeRowVector(ROW({}, {}), 1);
```

This test also doesn't need to create a 'tmp' table in DuckDB. Instead it can use a query like `VALUES (NULL)`;